src/command.js

```diff
@@ -2,8 +2,8 @@ import isWindows from 'is-windows'
 
 export default commandConvert
 
-const envUseUnixRegex = /\$(\w+)/ // $my_var
-const envUseWinRegex = /%(.*?)%/ // %my_var%
+const envUseUnixRegex = /\$(\w+)|\${(\w+)}/g // $my_var or ${my_var}
+const envUseWinRegex = /%(.*?)%/g // %my_var%
```

Comment:
The issue from my last PR arises from the use of `g` *and* `RegExp.prototype.exec`. So here it is OK to use `g` because `exec` is not used anymore, and the test I added ensures the problem does not return in the future. That said, I also think inlining would even safer, so I'll do that. Would you prefer that I keep them as variable inside `commandConvert` or that I inline them directly in the ternary on line 15?